tests/widgets_calendar.rs

```diff
@@ -11,6 +11,7 @@ use ratatui::{
 };
 use time::{Date, Month};
 
+#[track_caller]
```

Comment:
I just discovered this, why is it needed here?

Reply:
When the test fails, it should report the line of the failing test, and not the line in this method.
This makes things easier to fix when failures occur in this method.

Reply:
Alright interesting, thanks!